lib/components/user/monitored-trip/trip-summary-pane.js

```diff
@@ -1,41 +1,54 @@
-import React, { Component } from 'react'
+import React from 'react'
+import { FormattedList, FormattedMessage } from 'react-intl'
 
 import { dayFieldsToArray } from '../../../util/monitored-trip'
-import TripSummary from './trip-summary'
 
-class TripSummaryPane extends Component {
-  render () {
-    const { monitoredTrip } = this.props
-    const { itinerary } = monitoredTrip
+import TripSummary from './trip-summary'
 
-    if (!itinerary) {
-      return <div>No itinerary to display.</div>
-    } else {
-      // TODO: use the modern itinerary summary built for trip comparison.
+const BoldText = contents => <b>{contents}</b>
 
-      // For now, just capitalize the day fields from monitoredTrip.
-      const capitalizedDays = dayFieldsToArray(monitoredTrip).map(
-        day => `${day.charAt(0).toUpperCase()}${day.substr(1)}`
-      )
+/**
+ * Displays the summary information of a monitored trip.
+ */
+const TripSummaryPane = ({ monitoredTrip }) => {
+  const { itinerary, leadTimeInMinutes } = monitoredTrip
 
-      return (
-        <>
-          <TripSummary monitoredTrip={monitoredTrip} />
+  if (!itinerary) {
+    return (
+      <div>
+        <FormattedMessage id='common.itineraryDescriptions.noItineraryToDisplay' />
+      </div>
+    )
+  } else {
+    const days = (
+      <FormattedList
+        type='conjunction'
+        value={dayFieldsToArray(monitoredTrip).map(
+          d => <FormattedMessage id={`common.daysOfWeek.${d}`} key={d} />
+        )}
+      />
+    )
 
-          <p>
-            Happens on: <b>{capitalizedDays.join(', ')}</b>
-          </p>
-          <p>
-            Notifications:{' '}
-            <b>
-              {monitoredTrip.isActive
-                ? `${monitoredTrip.leadTimeInMinutes} min. before scheduled departure`
-                : 'Disabled'}
-            </b>
-          </p>
-        </>
-      )
-    }
+    return (
+      <>
+        {/* TODO: use the modern itinerary summary built for trip comparison. */}
+        <TripSummary monitoredTrip={monitoredTrip} />
+        <p>
+          <FormattedMessage
+            id='components.TripSummaryPane.happensOnDays'
+            values={{ b: BoldText, days }}
+          />
+        </p>
+        <p>
+          <FormattedMessage
+            id={`components.TripSummaryPane.notifications${
+              monitoredTrip.isActive ? '' : 'Disabled'
+            }`}
```

Comment:
Please do not create dynamic message IDs as it is hard to automatically keep track of which IDs are in the code. Instead, please create a separate variable for each possible message.